distribution/lib/Standard/Visualization/0.0.0-dev/src/Scatter_Plot.enso

```diff
@@ -100,19 +115,28 @@ type No_Fallback_Column
 ## PRIVATE
 
    Generates JSON that describes points data.
-Table.point_data : Vector
-Table.point_data self =
+Table.point_data : Vector -> Vector
+Table.point_data self all_fields =
     get_point_data field = field.lookup_in self . rename field.name . catch Any (_->Nothing)
     is_not_nothing x = case x of
         Nothing -> False
         _ -> True
-    columns = Point_Data.all_fields.map get_point_data . filter is_not_nothing
+    columns = all_fields.map get_point_data . filter is_not_nothing
     (0.up_to self.row_count).to_vector.map <| row_n->
         pairs = columns.map column->
             value = column.at row_n . catch_ Nothing
             [column.name, value]
         JS_Object.from_pairs pairs
 
+## PRIVATE
+
+   Generates JSON that describes points data.
+Table.multi_series_info : Number
+Table.multi_series_info self =
+    is_good c = c.is_numeric
+    columns = self.columns . filter is_good
```

Comment:
This is OK, but I imagine it would work just as well if it was
```suggestion
    columns = self.columns . filter .is_numeric
```
or
```suggestion
    columns = self.columns . filter c-> c.is_numeric
```

Reply:
I'm 100% for adding a specific name to a lambda so that the name serves as 'documentation' of the purpose of such function, but I'm not sure if that was adding clarity here. But that is a very minor thing and we can definitely keep the code as-is :) Sorry if I'm a bit too picky